homeassistant/components/mobile_app/webhook.py

```diff
@@ -295,10 +298,11 @@ async def webhook_render_template(hass, config_entry, data):
     resp = {}
     for key, item in data.items():
         try:
-            tpl = item[ATTR_TEMPLATE]
-            attach(hass, tpl)
+            tpl = template.Template(item[ATTR_TEMPLATE])
+            tpl.ensure_valid()
+            tpl.hass = hass
```

Comment:
```suggestion
            tpl = template.Template(item[ATTR_TEMPLATE], hass)
            tpl.ensure_valid()
```

Reply:
Actually probably don't need .ensure_valid() as it will be done when calling `async_render`